libcst/_nodes/statement.py

```diff
@@ -786,9 +786,13 @@ def _validate(self) -> None:
                 "Must use a Name node for AsName name inside ExceptHandler."
             )
         if self.type is not None and self.whitespace_after_except.empty:
-            raise CSTValidationError(
-                "Must have at least one space after except when ExceptHandler has a type."
-            )
+            # Space is only required when the first char in `type` could start
+            # an identifier.  In the most common cases, we want to allow
+            # grouping or tuple parens.
+            if isinstance(self.type, Name) and not self.type.lpar:
```

Comment:
you'll have to extract `self.type` to a local variable to make the type checker happy.